101-databricks-workspace-with-vnet-injection/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {},
+  "variables": {
+    "vnetName": "vnet-01",
+    "location": "West US",

```

Comment:
Add location parameter - see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#parameters

Otherwise if we have to change the region where we deploy samples, I suspect this will fail.